src/scenes/BROWSER/browserScene.ts

```diff
@@ -0,0 +1,121 @@
+
+import { getPageLoad } from './WebVitals/pageLoad';
+import { getWebVitals } from './WebVitals/webVitals';
+import { getDataTransferred } from './dataTransferred';
+import { getDistinctTargets } from './distinctTargets';
+import { getProbeDuration } from './probeDuration';
+
+export function getBrowserScene(
+  { metrics, logs, singleCheckMode }: DashboardSceneAppConfig,
+  checks: Check[] = [],
+  checkType: CheckType
+) {
+  return () => {
+    if (checks.length === 0) {
+      return getEmptyScene(checkType);
+    }
+    const timeRange = new SceneTimeRange({
+      from: 'now-1h',
+      to: 'now',
+    });
+    const { probe, job, instance } = getVariables(checkType, metrics, checks, singleCheckMode);
+    const variables = new SceneVariableSet({
+      variables: [probe, job, instance],
+    });
+
+    const minStep = getMinStepFromFrequency(checks?.[0]?.frequency);
+
+    const reachability = getReachabilityStat(metrics, minStep);
+    const uptime = getUptimeStat(metrics, minStep);
+
+    const distinctTargets = getDistinctTargets(metrics);
+    const probeDuration = getProbeDuration(metrics);
+    const editButton = getEditButton({ job, instance });
+    const annotations = getAlertAnnotations(metrics);
+
+    const webVitals = getWebVitals(metrics);
+    const pageLoad = getPageLoad(metrics);
+
+    return new EmbeddedScene({
+      $timeRange: timeRange,
+      $variables: variables,
+      $behaviors: [new behaviors.CursorSync({ key: 'sync', sync: DashboardCursorSync.Crosshair })],
+      $data: annotations,
+      controls: [
+        new VariableValueSelectors({}),
+        new SceneDataLayerControls(),
+        new SceneControlsSpacer(),
+        editButton,
+        new SceneTimePicker({ isOnCanvas: true }),
+        new SceneRefreshPicker({
+          intervals: ['5s', '1m', '1h'],
+          isOnCanvas: true,
+          refresh: '1m',
+        }),
+      ],
+      body: new SceneFlexLayout({
+        direction: 'column',
+        children: [
+          new SceneFlexLayout({
+            direction: 'row',
+            height: 150,
+            children: [new SceneFlexItem({ body: webVitals })],
+          }),
+          new SceneFlexLayout({
+            direction: 'row',
+            height: 200,
+            children: [new SceneFlexItem({ body: pageLoad })],
+          }),
+          new SceneFlexLayout({
+            direction: 'row',
+            height: 150,
+            children: [new SceneFlexItem({ body: uptime }), new SceneFlexItem({ body: reachability })],
+          }),
+          new SceneFlexLayout({
+            direction: 'row',
+            children: [getAssertionTable(logs, checkType, minStep)],
+          }),
+          new SceneFlexLayout({
+            direction: 'row',
+            height: 200,
+            children: [distinctTargets, probeDuration],
+          }),
+          getDataTransferred(metrics),
+          /* @todo: implement a table displaying web vitals metrics per url
```

Comment:
I was just thinking this would be a great addition.

Reply:
I'm working on this table, but I'm thinking to add it in a following PR as not to delay this first version ⭐️